fgrpc/grpcrunner.go

```diff
@@ -316,14 +316,26 @@ func grpcDestination(dest string) (parsedDest string) {
 func extractDialOptions(in metadata.MD) (out []grpc.DialOption) {
 	for k, v := range in {
 		switch k {
-		// Transfer these 2 and avoid having them duplicated in original MD
+		// Transfer these 2
 		case "user-agent":
-			delete(in, k)
 			out = append(out, grpc.WithUserAgent(v[0]))
 		case "host":
-			delete(in, k)
 			out = append(out, grpc.WithAuthority(v[0]))
```

Comment:
is this only these 2 that were missing on next dial because removed or something else?

Reply:
> is this only these 2 that were missing on next dial because removed or something else?

Yes, I don't think it's a good idea to modify metadata in Dial